util/redact/redact.go

```diff
@@ -0,0 +1,76 @@
+package redact
+
+import (
+	"encoding/hex"
+	"unsafe"
+
+	"go.uber.org/atomic"
+)
+
+// RedactLogEnabled defines whether the arguments of Error need to be redacted.
+var RedactLogEnabled atomic.String
+
+const (
+	RedactLogEnable  string = "ON"
+	RedactLogDisable string = "OFF"
+	RedactLogMarker  string = "MARKER"
+)
```

Comment:
this repo also include `pingcap/error` can we reuse them?

Reply:
I think the global variable cannot be reused if the `pingcap/error` is different. Let me reuse the constants.

Reply:
do you mean there be 2 different global variables?

go will choose only one larger version of pingcap/errors` when compile, so it should be only one

Reply:
For example, let's define "→" as "depends on":

```
A → B
B → C (v1, with variable c1)
A → C (v2, removed c1, add variable c2)
```

Do you mean only v2 is used? How about the access to `c1`?

Reply:
see https://go.dev/ref/mod#minimal-version-selection

Reply:
depends on the usage, it might report error on compile in your case

Reply:
https://github.com/pingcap/errors/pull/75